SpatialGDK/Documentation/content/cross-server-rpcs.md

```diff
@@ -0,0 +1,69 @@
+## Cross-server RPCs
+
+### Overview
+
+In Unreal’s native client-server architecture, your game server holds the canonical state of the whole game world. As there is a single game server, it has authority over all server Actors and so it is able to invoke and execute functions on Actors unhindered. 
+
+In SpatialOS games, there can be more than one server; these multiple servers are known as “server-workers”. (Find out more about server-workers as well as “client-workers” in the [glossary]({{URLRoot}}/content/glossary#workers).)  
+As a SpatialOS game runs across many server-workers, SpatialOS server-workers have the concept of “worker authority” - where only one server-worker at a time is able to invoke and execute functions on Actors. (Find out more about authority in the [glossary]({{URLRoot}}/content/glossary#authority).)
+
+As Unreal expects there to be only one server, rather than several servers, the GDK has a custom solution to take advantage of the SpatialOS distributed server architecture. This involves handling the scenario where a server-worker attempts to invoke an RPC on an Actor that another server-worker has authority over. This custom solution is the cross-server RPC. The GDK offers cross-server RPC in addition to support for the [native RPC types that Unreal provides (Unreal documentation)](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/RPCs).
+
+When a cross-server RPC is invoked by a non-authoritative server-worker, SpatialOS routes the execution through the SpatialOS [runtime]({{URLRoot}}/contents/glossary#runtime) to the authoritative server-worker - this authoritative server-worker executes the RPC.
```

Comment:
runtime -> Runtime

Reply:
@anne-edwards Are you sure that it should be capital R in the link? I thought the improbadoc links always were lower case?

Reply:
The instance of the word "Runtime" in square brackets should have a capital (because that's how we talk about "the SpatialOS Runtime", partly to distinguish it from something happening "at runtime"). But you're right, the word in the link itself should stay as it is :)

Reply:
Great, sorry misunderstood you updating now:)